src/backend/GnuCashBudget.Adapter.Abstractions/Models/AdapterResponse.cs

```diff
@@ -0,0 +1,7 @@
+namespace GnuCashBudget.Adapter.Abstractions.Models;
+
+public class AdapterResponse
+{
+    public string? ContinuationToken { get; set; }
+    public IEnumerable<Expense> Expenses { get; set; }
```

Comment:
Let's generalize this to `Transactions` or something

Reply:
done